payments-ui-core/build.gradle

```diff
@@ -30,15 +30,17 @@ dependencies {
     // Integration with observables
     implementation "androidx.compose.runtime:runtime-livedata:$androidxComposeVersion"
     implementation "com.google.accompanist:accompanist-flowlayout:$accompanistVersion"
-    // Tooling support (Previews, etc.)
-    debugImplementation "androidx.compose.ui:ui-tooling:$androidxComposeVersion"
+    implementation "androidx.compose.ui:ui-tooling-preview:$androidxComposeVersion"
```

Comment:
Is this necessary?

Reply:
Yes, I added some `@Preview` functions for live previewing in Android Studio, and ktlint fails without this dependency.